docs/clients.md

```diff
@@ -222,9 +222,9 @@ Use this module for sending user generated actions to Ultra’s analytics librar
 #### Add the dependency:
 ***With Node or for React Native:***
 
-Add the [latest alpha version of SDK]((https://www.npmjs.com/package/fk-platform-sdk/v/1.0.3-analytics-sdk-alpha)) to your package.json file:
+Add the [latest version of SDK](https://www.npmjs.com/package/fk-platform-sdk/v/1.0.5) to your `package.json` file:
```

Comment:
Latest version of SDK is 1.0.6 (and not 1.0.5). Please update.

Reply:
Done.